README.md

```diff
@@ -49,6 +49,7 @@ The image names don't map identically to the target names, to avoid conflicting
 | x86_64-apple-darwin                   | x86_64-apple-darwin-cross                   |
 | x86_64-pc-windows-msvc                | x86_64-pc-windows-msvc-cross                |
 | x86_64-unknown-linux-gnu              | x86_64-unknown-linux-gnu-sde-cross          |
+| mipsel-unknown-linux-uclibc           | /mipsel-unknown-linux-uclibc-cross          |
```

Comment:
Just needs to remove the leading `/` in `/mipsel-unknown-linux-uclibc-cross`.

Reply:
:sob: That's my bad, updating now.

> I will be giving you co-author credit for that file btw. Once I make my commits, feel free to squash the commits to remove my commits from the history (you don't need to credit me).

I see no reason to do that, you absolutely guided me through this!

Reply:
> 😭 That's my bad, updating now.
> 
> > I will be giving you co-author credit for that file btw. Once I make my commits, feel free to squash the commits to remove my commits from the history (you don't need to credit me).
> 
> I see no reason to do that, you absolutely guided me through this!

You did a substantial amount of work and I used it. That deserves credit.